### PR TITLE
US12052: Publication Templating System

### PR DESCRIPTION
**Pull request by lukemoderwell**
This PR:
  - defines the idea of a publication or a generic syndicated piece of content that could be an article, video, podcast, or other.
  - encapsulates shared and medium-specific styles into `_publication.scss`
  - sets up dynamic templates that render based on some simple logic from the URL string
  - refactors individual templates, views, and controllers into singular publication concept with partials for media-type templates.

Note: no unit tests yet. Maybe I should have them?

### CODE REVIEW

File: apps/crossroads_interface/web/controllers/publications_controller.ex [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  alias CrossroadsInterface.Plug
+  alias CrossroadsInterface.NotfoundController
+
+  plug Plug.Meta
+  plug Plug.ContentBlocks
+  plug Plug.BodyClass, "crds-styles"
+
+  def index do
+  end
+
```
Comment (richardbarker) [inline apps/crossroads_interface/web/controllers/publications_controller.ex]: This is called for 2 routes but not doing anything.  We should either use show for all cases or implement this.

File: apps/crossroads_interface/web/router.ex [hunk 2]
```diff
@@ -26,6 +26,9 @@ defmodule CrossroadsInterface.Router do
     get "/groups/search/*path", CrdsGroupsController, :index
     get "/srfp/*path", CrdsSrfpController, :index
     get "/series/:id/*path", CmsSeriesController, :show
+    get "/publications/:medium/:id/*path", PublicationsController, :show
+    get "/publications/:medium", PublicationsController, :index
+    get "/publications/", PublicationsController, :index
```
Comment (richardbarker) [inline apps/crossroads_interface/web/router.ex]: Not sure if this should include trailing slash (others don't).  
Also see previous comment about :index

**Comment by richardbarker** [comment]
Do we want to check in full static pages in the templates?  I understand putting something out there as until we get the service working that will pull content in.  However, I think maybe it would be more helpful to actually build the templates out of fake content values.

File: apps/crossroads_interface/web/router.ex (hunk 2 above)
Comment (richardbarker) [reply apps/crossroads_interface/web/router.ex]: Unit tests needed

File: apps/crossroads_interface/config/dev.exs [hunk 3]
```diff
@@ -42,7 +42,9 @@ config :crossroads_interface, CrossroadsInterface.Endpoint,
   ]
 
 # Do not include metadata nor timestamps in development logs
-config :logger, :console, format: "[$level] $message\n", level: :debug
+config :logger,
```
Comment (mjvcallibrity) [inline apps/crossroads_interface/config/dev.exs]: This should be reverted back to how it was before this work was done.

**Pull request merged**